build/charts/antrea/conf/antrea-controller.conf

```diff
@@ -74,6 +74,10 @@ tlsCipherSuites: {{ .Values.tlsCipherSuites | quote }}
 # TLS min version from: VersionTLS10, VersionTLS11, VersionTLS12, VersionTLS13.
 tlsMinVersion: {{ .Values.tlsMinVersion | quote }}
 
+# File path of the certificate bundle for all the signers that is recognized for incoming client
+# certificates.
+apiserverClientCAFile: {{ .Values.apiserverClientCAFile | quote }}
```

Comment:
Like we don't name the other similar configurations apiserverTLSMinVersion, it could just be `clientCAFile`, I think there is no ambiguity that this is for client authentication.

Reply:
updated.